sql/moz-fx-data-shared-prod/mozilla_vpn_derived/funnel_product_page_to_subscribed_v1/query.sql

```diff
@@ -20,10 +20,65 @@ WITH stripe_plans AS (
 ),
 events AS (
   SELECT
-    *,
     DATE(`timestamp`) AS partition_date,
+    `timestamp` receiveTimestamp,
+    flow_id,
+    entrypoint,
+    entrypoint_experiment,
+    entrypoint_variation,
+    utm_term,
+    utm_source,
+    utm_medium,
+    utm_campaign,
+    utm_content,
+    ua_version,
+    ua_browser,
+    plan_id,
+    promotion_code,
+    oauth_client_id,
+    logger,
+    event_type,
+    app_version,
+    os_name,
+    os_version,
+    country,
+    LANGUAGE,
+    user_id,
+    utm_term,
+    utm_source,
+    utm_medium,
+    utm_campaign,
+    utm_content,
+    ua_version,
+    ua_browser,
+    entrypoint,
+    entrypoint_experiment,
+    entrypoint_variation,
+    flow_id,
+    service,
+    email_type,
+    email_provider,
+    oauth_client_id,
+    connect_device_flow,
+    connect_device_os,
+    sync_device_count,
+    sync_active_devices_day,
+    sync_active_devices_week,
+    sync_active_devices_month,
+    email_sender,
+    email_service,
+    email_template,
+    email_version,
+    plan_id,
+    product_id,
+    promotion_code,
+    payment_provider,
+    checkout_type,
+    source_country,
```

Comment:
* `` `timestamp` receiveTimestamp`` looks like a mistake, as `timestamp` ends up aliased as `receiveTimestamp` and there's no column named `timestamp` in the output, so downstream references to `events.timestamp` will fail.
* A number of columns are being selected twice (e.g. `flow_id`, `entrypoint`).
* A number of columns being selected aren't actually used here (e.g. `logger`, `app_version`, `language`).
```suggestion
    `timestamp`,
    flow_id,
    entrypoint,
    entrypoint_experiment,
    entrypoint_variation,
    utm_term,
    utm_source,
    utm_medium,
    utm_campaign,
    utm_content,
    ua_version,
    ua_browser,
    plan_id,
    promotion_code,
    oauth_client_id,
    event_type,
    os_name,
    os_version,
    country,
    user_id,
    service,
    product_id,
```